secretsauce/apps/portal/models.py

```diff
@@ -113,3 +120,37 @@ class ConstraintParameterRelationship(models.Model):
         on_delete=models.CASCADE,
     )
     coefficient = models.FloatField()
+
+class Schema(models.Model):
```

Comment:
1. Maybe can change the name of `Schema` to `HeaderList`? I thought schema is quite a generic word referring to a typical database structure and does not provide much contextual meaning
2. I find the term `ItemList` a little odd semantically like when you use it as a view i.e. a list's list--- `class ItemListList(generics.ListCreateAPIView):`. Cant think of a better name too. If you cant think of one, I guess we can leave it the same

Reply:
Sure, will make the changes.